docs/_documentations/mdt-che-overview.md

```diff
@@ -20,16 +20,5 @@ The Eclipse tools are [open source](https://github.com/eclipse/codewind-eclipse)
 
 To get started, see [Getting started with Codewind](mdteclipsegettingstarted.html).
 
-To uninstall Codewind, see [Uninstalling Codewind for Eclipse](mdteclipseuninstall.html)
+To uninstall Codewind, see [Uninstalling Codewind for Eclipse](mdteclipseuninstall.html).
```

Comment:
I am thinking of the overview page of `Codewind for Eclipse Che` should be similar to the overview page of `Codewind for VS Code` here: https://www.eclipse.org/codewind/mdt-vsc-overview.html. So basically it should include introduction and features summary.

Reply:
Introduction I've left as is, added the **Features** section from VS Code modified for Eclipse.
Three questions:

1. **Features** in VS Code links to a commands page which we don't have for Eclipse, so I've omitted the link.  Do we need such a page for Eclipse? I've scanned the help for commands, can't find any, so I guess not? 
2. Is the following modified sentence correct for Eclipse?
   - View application and build logs in the Eclipse **Output** view.
3. Is the following workspace sentence still valid?
   - Scope your Eclipse workspace to a single project or to your *codewind-workspace*.

Reply:
Resolved these points